test/ExecutionFramework.h

```diff
@@ -279,7 +280,13 @@ class ExecutionFramework
 	solidity::frontend::RevertStrings m_revertStrings = solidity::frontend::RevertStrings::Default;
 	solidity::frontend::OptimiserSettings m_optimiserSettings = solidity::frontend::OptimiserSettings::minimal();
 	bool m_showMessages = false;
+
+	EVMHost* m_evmcHost;
+
 	std::shared_ptr<EVMHost> m_evmHost;
+	std::shared_ptr<EVMHost> m_ewasmHost;
```

Comment:
We call `reset` all the time on this - does it make sense to keep both around?

Reply:
I was not sure where you exactly referring to. We have two real evm host instances, one for `evm` and one for `ewasm`. And one where `reset` is called `m_evmcHost`. This is just a pointer that may point either to the `evm` instance or to the `ewasm` instance.

Reply:
I think my question was whether we should not have just the single member `unique_ptr<EVMHost> m_evmcHost` and recreate it whenever we would call reset.

Reply:
implemented the recreation in`ExecutionFramework::setEvm(..)`.